File: Backend/API/Group/DTO/UpdateGroupDto.cs
using System.ComponentModel.DataAnnotations;

namespace API.Group.DTO
{
    public record UpdateGroupDto
    {
        [StringLength(50)]
        public string? Name { get; init; }

        [StringLength(400)]
        public string? Description { get; init; }

        public Guid? OwnerId { get; set; }
    }
}

File: po-rowno-app/src/app/_services/group/dto/update-group.dto.ts
export interface UpdateGroupDto {
  /**
   * @minLength 0
   * @maxLength 50
   */
  name?: string | null;
  /**
   * @minLength 0
   * @maxLength 400
   */
  description?: string | null;
  /** @format uuid */
  ownerId?: string | null;
}
